### PR TITLE
VB-681: Visit cancellation reason

### PR DESCRIPTION
**Pull request by tpmcgowan**
* Add visit cancellation reason page at `/visit/:reference/cancel`
* Validate cancellation reason and update the booking in the visit scheduler
* After successful cancellation, redirect to `/visit/cancelled` (a placeholder page - to be done as VB-682)

### CODE REVIEW

File: server/views/pages/visit/cancelConfirmation.njk [hunk 1]
```diff
@@ -0,0 +1 @@
+CANCELLATION CONFIRMATION!
```
Comment (tpmcgowan) [inline server/views/pages/visit/cancelConfirmation.njk:1]: Placeholder template - to be sorted with VB-682

File: server/services/visitSessionsService.ts [hunk 2]
```diff
@@ -191,6 +191,23 @@ export default class VisitSessionsService {
...
+    outcome,
+  }: {
+    username: string
+    reference: string
+    outcome: OutcomeDto
+  }): Promise<Visit> {
+    const token = await this.systemToken(username)
+    const visitSchedulerApiClient = this.visitSchedulerApiClientBuilder(token)
+
+    const visit = await visitSchedulerApiClient.cancelVisit(reference, outcome)
```
Comment (psoleckimoj) [inline server/services/visitSessionsService.ts:206]: I think if you do a return, you can also remove the `await` so `return visitSched.....` ? Less lines 😄

Comment (tpmcgowan) [reply server/services/visitSessionsService.ts]: Good spot! Fixed

**Review by psoleckimoj — APPROVED** [review]
LGTM!

**Pull request merged**